pkg/sosreport/run-sosreport.sh

```diff
@@ -0,0 +1,7 @@
+#! /bin/sh
```

Comment:
This is a script that traverses the network each time and doesn't get minified. I don't think it needs a license.